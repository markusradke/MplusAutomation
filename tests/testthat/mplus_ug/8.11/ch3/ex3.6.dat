    0.000000     0.573051    -0.175230
    1.000000     1.090042    -0.115272
    0.000000     0.179867     1.088520
    2.000000    -0.694153    -0.766538
    2.000000    -1.367410    -0.050084
    2.000000     0.579605    -0.658335
    1.000000     0.463916    -0.898300
    2.000000     0.180512     0.113829
    2.000000     0.253673    -1.341994
    1.000000     1.027419     0.677408
    1.000000    -0.759287    -0.447889
    2.000000     1.104495    -0.507631
    1.000000     0.761720    -1.901134
    2.000000     2.318569    -2.272791
    2.000000    -0.846025     0.155868
    1.000000    -0.295120     0.881524
    1.000000     1.055233     0.107389
    0.000000     0.508650     0.206923
    1.000000    -0.805411     0.766234
    2.000000     0.510023     0.068812
    2.000000    -2.154671    -0.286850
    1.000000     2.345841     0.284032
    1.000000    -1.172163    -1.161900
    1.000000    -0.271187    -1.231314
    0.000000     0.989511     2.269332
    1.000000     1.454726    -2.975978
    0.000000     0.156104     1.350790
    1.000000    -0.180285     0.271133
    0.000000     0.683911     0.292498
    0.000000    -2.090380     1.391452
    2.000000     0.446963     0.080041
    2.000000    -0.567542     0.144373
    1.000000     0.855505    -0.016467
    1.000000    -0.064293    -0.935022
    2.000000    -2.464700    -0.408491
    2.000000    -0.469326    -0.416403
    2.000000    -0.090957    -0.097848
    2.000000    -0.392073     0.245876
    1.000000    -0.478192     0.888866
    2.000000    -1.888729    -1.007475
    1.000000    -0.582148     0.048988
    2.000000    -0.124607     0.848458
    0.000000     0.034340     0.381900
    2.000000    -1.228212    -0.728623
    2.000000    -0.597038    -1.078322
    1.000000     0.299540     0.482481
    1.000000    -1.355690    -0.159522
    1.000000     1.588814    -1.132670
    1.000000     1.380470    -0.402537
    0.000000     1.688172     0.296591
    1.000000     0.489773     1.324936
    2.000000    -2.813280     0.163674
    0.000000    -0.730827     0.103428
    2.000000     2.259616    -1.067363
    1.000000     0.144324     2.007878
    1.000000     0.005238    -0.808209
    1.000000     0.302495     0.127302
    2.000000    -1.452235    -2.448208
    1.000000    -0.855961    -0.702519
    0.000000    -1.644768     1.776425
    1.000000     1.007635    -0.220893
    2.000000     0.045692    -0.080164
    1.000000    -1.139363    -0.057090
    1.000000    -1.154268    -0.833996
    2.000000    -0.354129    -0.446513
    2.000000    -0.148284    -1.838382
    2.000000    -0.922427    -0.239804
    2.000000     1.660074     0.240847
    2.000000    -1.078953    -0.180804
    1.000000     0.211655    -0.799270
    2.000000    -1.254994    -0.167905
    1.000000    -1.110931    -1.695846
    0.000000     2.625872     1.929568
    0.000000    -1.080526    -0.517819
    0.000000     0.762969     1.781710
    2.000000    -0.101754    -1.519739
    2.000000    -1.122865    -0.117698
    2.000000    -0.894718    -1.187673
    2.000000    -0.213364     0.204102
    0.000000    -1.065222     0.464775
    1.000000    -0.387833    -0.096412
    0.000000     0.250665     0.868009
    1.000000    -1.010871     2.618576
    1.000000     0.527030    -0.038221
    2.000000    -1.373059     0.165736
    2.000000     0.520681    -1.163071
    2.000000    -1.605275    -0.274087
    0.000000     2.103710     1.852987
    2.000000     0.354805    -0.522118
    0.000000     0.100772    -0.102803
    2.000000    -1.704053     0.312722
    1.000000     1.311074    -0.150245
    0.000000    -0.196207     0.215611
    0.000000     0.779012     0.533775
    2.000000     0.899194    -0.510032
    0.000000     1.313883    -0.605905
    1.000000    -0.895761     1.050430
    1.000000    -0.183562     0.582855
    1.000000    -1.307288     0.100200
    2.000000     1.991547    -1.139637
    2.000000     1.200176    -0.641724
    2.000000    -0.466598     0.269281
    0.000000     1.063671     1.307742
    1.000000    -0.521088     0.137343
    0.000000     0.285447     0.605200
    2.000000     0.542719    -1.251934
    2.000000     0.530206     0.034122
    1.000000    -0.003589    -0.625362
    0.000000     0.157191     0.417853
    0.000000    -0.283336     1.267245
    0.000000    -0.848835     1.851695
    1.000000    -1.662003    -0.626968
    1.000000     0.330181    -1.946720
    1.000000     0.327435     0.204817
    2.000000    -1.162120    -0.074466
    1.000000    -0.590387    -0.568837
    2.000000    -1.700674    -0.744056
    0.000000     0.997753     0.159104
    0.000000    -0.365480     1.336717
    0.000000    -0.182482     1.107860
    2.000000     0.637273    -1.250547
    1.000000    -0.282680    -0.964912
    0.000000     0.700194    -0.573142
    0.000000     0.887330     0.251705
    1.000000    -0.352938     0.234585
    2.000000    -0.631500    -0.238242
    1.000000    -1.429091    -1.142571
    0.000000     1.042027     0.220666
    2.000000     0.474035    -0.082168
    1.000000    -1.173233     1.024470
    0.000000     1.480269     0.082213
    2.000000     0.988791    -0.318939
    0.000000    -0.632654     1.576646
    0.000000     0.735089     0.718696
    2.000000    -0.217398    -1.015878
    1.000000     0.261811    -0.147346
    2.000000    -0.000443     0.254631
    0.000000     0.787727     0.492485
    2.000000     0.554994    -1.134404
    0.000000    -0.179577     0.309703
    0.000000    -0.625061     0.972481
    1.000000     1.496123     0.910772
    1.000000    -0.154379     0.441979
    1.000000    -0.410498    -0.099287
    0.000000     0.148157     1.005702
    2.000000     1.231496     0.049212
    2.000000     0.236292    -1.438562
    1.000000     0.948056    -1.021500
    0.000000    -0.024148     0.582766
    1.000000    -0.757783     0.883052
    0.000000     0.438144     0.541813
    1.000000     0.341296    -0.501303
    2.000000    -2.006273    -0.681526
    2.000000    -0.432574    -0.374755
    1.000000     0.929534    -1.031156
    1.000000     0.646291    -0.440813
    1.000000     0.527831    -0.008297
    1.000000    -0.084564    -0.378982
    1.000000     0.630619     1.288976
    0.000000     1.043301     0.806248
    1.000000     0.518950     0.309523
    2.000000    -1.742697    -0.553668
    2.000000    -1.258468     0.349651
    2.000000    -0.359877    -1.788849
    1.000000     0.456252     0.301679
    2.000000     0.823873    -1.457744
    1.000000    -0.482347     0.446204
    0.000000     0.520741     0.946513
    2.000000     0.920158     0.568478
    2.000000     0.008103    -0.436071
    1.000000    -0.287143    -0.605594
    1.000000     0.161226    -0.926396
    2.000000    -1.648644    -1.392965
    2.000000     0.692887    -0.879911
    0.000000     0.358199     1.131302
    0.000000     1.121815     1.281492
    2.000000     0.324771    -1.330312
    0.000000     0.841865     0.824594
    2.000000    -1.787104     1.069722
    2.000000    -0.647019    -1.002748
    0.000000    -1.049286     0.853345
    1.000000    -1.370628     1.732520
    0.000000     1.013821     1.348738
    1.000000    -0.168757    -0.601257
    2.000000    -3.138746    -1.538631
    0.000000     0.646988     1.287929
    0.000000     1.521697     0.158848
    1.000000     0.495691     1.149072
    1.000000    -0.882568    -0.534994
    2.000000     0.080439    -0.731038
    1.000000    -2.125611    -0.893665
    2.000000     0.713350    -1.887653
    2.000000     0.122162     0.111848
    1.000000    -1.191328    -0.947461
    1.000000     0.632221     0.753587
    2.000000     0.677648    -1.706342
    1.000000    -1.077849     0.179615
    1.000000    -0.225950     0.627576
    2.000000    -0.147412    -1.230579
    1.000000    -0.103803    -0.542742
    1.000000     0.854455    -0.798283
    1.000000    -0.083330    -0.610441
    0.000000     1.209175     1.676376
    1.000000    -0.495295     0.889091
    2.000000     0.454753    -1.821117
    2.000000    -1.368259     0.442774
    0.000000     0.293520     0.817096
    2.000000    -3.109075     0.011060
    1.000000     0.044320    -0.033948
    1.000000     1.381966     0.377715
    1.000000    -0.270848     0.953480
    0.000000    -0.913053     0.808203
    1.000000     1.363720    -1.217698
    1.000000    -0.844085     0.253331
    2.000000     0.037702     0.068187
    1.000000    -0.652522     0.564978
    2.000000     1.528843    -2.016222
    2.000000    -1.547121    -1.879351
    0.000000    -0.279376     1.435888
    0.000000     1.482167     0.329410
    1.000000     1.026840     0.632514
    1.000000    -0.042544     0.261180
    0.000000    -0.306122     0.440869
    1.000000     0.208646     0.062846
    2.000000    -1.437663    -1.207147
    0.000000     0.817268     0.704504
    2.000000    -0.280294    -0.886381
    1.000000    -0.637129     0.573238
    0.000000    -1.915427     0.954961
    0.000000    -0.199550     0.623056
    1.000000     1.339256    -2.440599
    2.000000     0.603255    -0.549585
    1.000000     0.953121    -0.026660
    1.000000    -0.083851    -0.385483
    2.000000     2.143761    -1.033161
    1.000000    -0.842728    -0.154938
    1.000000    -0.006394     0.703353
    0.000000     1.073031     0.886030
    2.000000    -0.833043     1.327914
    2.000000    -0.238606    -1.026772
    1.000000     0.379434     0.722030
    0.000000     1.041775     0.506457
    1.000000     0.407309    -1.538500
    0.000000     0.698800     1.790311
    2.000000     0.894190    -1.302233
    2.000000     1.167797    -1.155155
    0.000000    -0.462345     1.531175
    1.000000    -1.039146    -0.035003
    0.000000    -0.708254     0.862989
    2.000000     0.386628    -1.451426
    1.000000     1.289453     0.768371
    2.000000    -0.902746    -0.864477
    1.000000     0.004397     0.684072
    1.000000     1.309255    -0.471564
    0.000000     2.150614    -0.737749
    1.000000    -0.844899     0.739691
    2.000000    -0.727949    -0.725231
    1.000000     1.733230    -0.442936
    0.000000     0.423892     1.655431
    1.000000    -1.322885     1.315887
    0.000000    -0.603284     0.259655
    0.000000    -1.444013     0.874422
    2.000000     0.848341    -0.326220
    1.000000    -0.199808    -0.671324
    2.000000     1.212288    -0.314465
    1.000000     0.718520     0.085917
    2.000000    -2.141871     0.285501
    2.000000    -0.675669     0.485942
    0.000000     0.409421     2.857476
    0.000000     3.251647     1.188059
    0.000000    -0.518283    -0.272866
    0.000000    -1.719268     0.707061
    1.000000     0.797133    -0.132003
    2.000000     0.601180     0.160395
    0.000000    -1.791355     2.094605
    0.000000    -0.101720     2.307213
    0.000000    -0.238676     0.888635
    2.000000    -0.529308    -1.094086
    1.000000    -0.598956     0.268941
    0.000000     1.634677     1.103158
    2.000000    -0.920681    -0.843669
    2.000000     0.307034    -0.537980
    1.000000    -0.452624     0.560690
    1.000000     0.823038    -0.608066
    2.000000     0.337933    -0.831284
    2.000000    -0.241834    -1.021001
    2.000000    -0.576512     0.425300
    1.000000    -0.017298    -0.698727
    0.000000     2.483435     1.120482
    2.000000     0.337089     0.162480
    2.000000     0.526899     0.909041
    1.000000     0.481765    -1.021191
    2.000000    -0.699131    -1.938625
    1.000000     1.446890    -1.205743
    1.000000    -0.273884    -0.803969
    2.000000    -0.010319    -1.360551
    2.000000     0.397331    -3.145148
    1.000000     1.306586    -0.852620
    2.000000    -0.685945    -1.054061
    2.000000    -0.771602    -2.611961
    0.000000     1.679024    -0.085706
    1.000000     0.897929    -0.869805
    2.000000     0.972409     0.555395
    1.000000    -1.337919     0.429926
    1.000000    -0.357646    -0.932405
    2.000000    -0.014553    -0.778397
    2.000000    -1.070475    -1.612643
    2.000000    -0.038405    -0.286104
    2.000000    -1.614130     0.353920
    2.000000     1.409971    -0.600984
    2.000000    -0.309985     0.184597
    1.000000    -0.822120     1.682930
    0.000000     0.604875    -0.589871
    2.000000    -2.024955    -0.437743
    1.000000     0.649338    -1.444095
    1.000000    -0.124574     0.611455
    2.000000    -0.450624    -0.289032
    2.000000     0.046881    -0.758778
    0.000000     1.408185     1.043946
    1.000000     0.025235    -0.353672
    2.000000     0.111081     0.985010
    2.000000    -1.156996    -0.989380
    2.000000    -1.191645    -0.113412
    2.000000    -0.725523     0.049068
    1.000000     0.886483     0.560904
    2.000000    -0.468871    -1.206464
    1.000000    -0.763727    -0.345737
    0.000000     1.883960     0.766058
    0.000000    -0.140322     0.394793
    1.000000     1.406439     1.129451
    2.000000    -1.909077    -0.620734
    1.000000    -0.227332     1.662168
    1.000000     0.231743     0.100638
    2.000000    -0.156710    -2.364484
    1.000000     0.647727     0.369044
    2.000000    -0.107720    -2.234895
    2.000000     1.001601    -1.425300
    0.000000    -0.438981     0.865333
    2.000000     0.786318    -1.410603
    0.000000     1.151548     0.876034
    2.000000    -0.601684    -0.071130
    1.000000     1.336016     0.629972
    2.000000    -0.478344     0.640302
    1.000000    -1.065090    -0.296255
    2.000000     0.888141     0.142590
    1.000000    -0.079587     0.410057
    1.000000    -0.864393     0.673658
    0.000000     0.023418     0.223939
    1.000000    -0.968080     0.882103
    1.000000     1.452007     1.015440
    2.000000     1.313348     0.237363
    2.000000     0.146615    -1.620969
    0.000000     1.921933    -0.303637
    2.000000     0.123814     0.376483
    2.000000    -0.596481    -0.037010
    0.000000    -0.647430     0.044437
    2.000000    -2.167467    -0.343634
    2.000000    -0.716694    -0.995137
    1.000000     0.921659    -0.186363
    2.000000    -0.828327    -0.856475
    2.000000     0.893144    -0.439566
    2.000000     1.220192     0.308603
    0.000000     1.593919     0.495517
    2.000000     0.048602     0.054327
    0.000000    -0.047091     1.252484
    0.000000     1.049187     0.547750
    1.000000    -1.679494    -0.152874
    2.000000     0.268712    -0.402545
    2.000000     0.735202    -1.074128
    1.000000     0.548786    -0.404752
    2.000000    -0.588534    -0.758553
    2.000000    -2.491072    -0.661860
    1.000000    -1.123784    -0.557013
    1.000000     1.440759     1.177360
    1.000000     0.671284    -0.260393
    1.000000    -0.002730     0.122410
    0.000000     0.139407     0.212248
    2.000000    -0.355955    -1.633714
    1.000000     0.080184    -1.015845
    1.000000    -0.191786     1.689297
    0.000000     2.349897     1.113138
    1.000000    -1.628909    -0.200864
    2.000000    -1.152455    -0.612066
    2.000000    -1.942149    -0.017788
    1.000000    -0.458894    -0.143755
    0.000000     0.445642     1.083808
    2.000000    -0.237936    -0.816727
    2.000000     2.545458     1.364745
    2.000000     0.328073    -1.396832
    1.000000    -0.945236    -0.441755
    1.000000    -1.469622     0.302606
    2.000000     0.444912     0.485789
    2.000000     1.079312    -0.798910
    2.000000    -1.099811    -2.004489
    1.000000     1.194329    -0.143110
    1.000000    -0.480040    -0.227389
    2.000000     0.542561    -2.515656
    2.000000    -0.101767     0.639474
    0.000000    -0.159058     2.832829
    1.000000     0.027494    -0.651239
    2.000000     1.117207     1.027536
    0.000000    -0.150161     0.315622
    1.000000     0.182592    -0.119080
    2.000000    -0.317180    -0.186164
    1.000000    -0.046815     0.887718
    1.000000    -0.825881     1.076636
    2.000000    -1.076124     0.164244
    2.000000    -1.432209    -1.408324
    0.000000     2.938392     0.352521
    1.000000    -2.251692    -0.718311
    1.000000    -0.021161    -0.577073
    0.000000     0.204399     0.126607
    2.000000    -0.516106    -1.600714
    2.000000     0.234564    -0.806120
    0.000000    -0.215019     0.678247
    2.000000    -0.610119    -0.775596
    1.000000     0.976169    -0.678995
    2.000000     0.176545    -2.197861
    1.000000    -1.489876    -0.081979
    1.000000    -0.410390     0.071516
    2.000000    -1.045586    -0.423820
    1.000000     1.769768     0.313457
    1.000000     0.850608    -0.831174
    1.000000    -0.485545     1.833271
    2.000000     0.384164    -0.460128
    1.000000    -0.596864     0.610095
    0.000000    -0.248228     1.213070
    1.000000     0.474596     0.177468
    1.000000    -0.731344    -0.596928
    0.000000    -0.329338     0.300786
    0.000000     0.319225     0.549400
    2.000000    -0.831143    -1.273211
    1.000000     0.374481     1.481391
    2.000000     0.760475    -1.182282
    0.000000     1.231722     0.697808
    2.000000     0.894761    -0.732890
    2.000000     1.578060    -1.819783
    0.000000     0.785122     0.116707
    2.000000    -0.000812    -0.451043
    1.000000    -0.540853     0.486724
    0.000000    -0.575000     1.116990
    1.000000     1.108620    -1.797010
    0.000000     0.107540     1.392949
    2.000000    -0.208100    -0.174168
    0.000000     0.151135     0.263803
    1.000000    -0.328900    -0.528071
    0.000000    -1.972865     1.683448
    0.000000     0.648978     0.052686
    1.000000    -0.950877     0.198332
    2.000000    -0.853172    -0.508668
    1.000000    -0.704313     0.430406
    1.000000    -0.693137     0.487261
    1.000000     0.088182    -0.155483
    2.000000     0.686391    -2.110151
    1.000000     0.637375    -0.745124
    0.000000     1.316737     0.305113
    1.000000     0.007940     0.740636
    2.000000     0.395153     0.161268
    2.000000    -2.218271    -0.709433
    1.000000     1.447092    -1.968813
    1.000000    -0.188381    -0.083253
    2.000000    -0.364539    -0.788623
    2.000000    -1.774245     0.535814
    1.000000    -1.268211    -0.177412
    0.000000    -0.948668     0.765383
    0.000000     1.372770     2.161662
    2.000000     1.326410    -1.713818
    2.000000    -0.499960    -0.980734
    1.000000    -1.858780    -1.002812
    1.000000    -0.617675    -1.295669
    2.000000     1.464860    -1.505978
    1.000000     0.180059     0.222230
    2.000000     0.749339    -0.986275
    0.000000     0.435088     1.137152
    2.000000    -0.463605    -0.706667
    0.000000    -0.554756     0.354948
    0.000000    -0.486702     2.011997
    0.000000     0.273762     1.301096
    2.000000     0.594459    -0.238691
    0.000000     1.099579     0.245759
    2.000000    -0.231304    -0.143851
    1.000000    -0.841948    -0.732580
    1.000000    -0.484377     0.683601
    2.000000    -1.249850    -0.698791
    2.000000    -0.248540    -1.488241
    2.000000     0.215300     0.609386
    0.000000     1.162643    -0.588019
    1.000000    -1.884015     1.358795
    0.000000    -0.157527    -0.568032
    1.000000     0.738848     0.031132
    2.000000    -0.221319     0.658794
    2.000000    -0.095925    -1.819444
    2.000000     0.314766    -0.311164
    0.000000    -0.541460     1.155530
    2.000000     1.149248    -1.001744
    1.000000     1.896521     1.381426
    0.000000    -0.302040     0.067875
    1.000000    -0.503242    -0.483770
    0.000000     1.887076     0.794827
    2.000000    -0.178094     0.368384
