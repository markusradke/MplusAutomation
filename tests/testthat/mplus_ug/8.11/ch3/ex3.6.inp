TITLE:	this is an example of a multinomial
        logistic regression for an unordered
        categorical (nominal) dependent variable
        with two covariates
DATA:	FILE IS ex3.6.dat;
VARIABLE:	NAMES ARE u1 x1 x3;
	NOMINAL IS u1;
MODEL:  u1 ON x1 x3;
SAVEDATA:
	H5RESULTS = ex3.6.h5;
OUTPUT:
	CINTERVAL STANDARDIZED;