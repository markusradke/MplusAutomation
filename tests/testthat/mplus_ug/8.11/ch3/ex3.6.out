Mplus VERSION 8.11 (Mac)
MUTHEN & MUTHEN
05/24/2024  12:05 PM

INPUT INSTRUCTIONS

  TITLE:	this is an example of a multinomial
          logistic regression for an unordered
          categorical (nominal) dependent variable
          with two covariates
  DATA:	FILE IS ex3.6.dat;
  VARIABLE:	NAMES ARE u1 x1 x3;
  	NOMINAL IS u1;
  MODEL:  u1 ON x1 x3;
  SAVEDATA:
  	H5RESULTS = ex3.6.h5;
  OUTPUT:
  	CINTERVAL STANDARDIZED;



INPUT READING TERMINATED NORMALLY



this is an example of a multinomial
logistic regression for an unordered
categorical (nominal) dependent variable
with two covariates

SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         500

Number of dependent variables                                    1
Number of independent variables                                  2
Number of continuous latent variables                            0

Observed dependent variables

  Unordered categorical (nominal)
   U1

Observed independent variables
   X1          X3


Estimator                                                      MLR
Information matrix                                        OBSERVED
Optimization Specifications for the Quasi-Newton Algorithm for
Continuous Outcomes
  Maximum number of iterations                                 100
  Convergence criterion                                  0.100D-05
Optimization Specifications for the EM Algorithm
  Maximum number of iterations                                 500
  Convergence criteria
    Loglikelihood change                                 0.100D-02
    Relative loglikelihood change                        0.100D-05
    Derivative                                           0.100D-02
Optimization Specifications for the M step of the EM Algorithm for
Categorical Latent variables
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
Optimization Specifications for the M step of the EM Algorithm for
Censored, Binary or Ordered Categorical (Ordinal), Unordered
Categorical (Nominal) and Count Outcomes
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-02
  Basis for M step termination                           ITERATION
  Maximum value for logit thresholds                            15
  Minimum value for logit thresholds                           -15
  Minimum expected cell size for chi-square              0.100D-01
Optimization algorithm                                         EMA
Integration Specifications
  Type                                                    STANDARD
  Number of integration points                                  15
  Dimensions of numerical integration                            0
  Adaptive quadrature                                           ON
Cholesky                                                       OFF

Input data file(s)
  ex3.6.dat
Input data format  FREE


UNIVARIATE PROPORTIONS AND COUNTS FOR CATEGORICAL VARIABLES

    U1
      Category 1    0.242          121.000
      Category 2    0.368          184.000
      Category 3    0.390          195.000



UNIVARIATE SAMPLE STATISTICS


     UNIVARIATE HIGHER-ORDER MOMENT DESCRIPTIVE STATISTICS

         Variable/         Mean/     Skewness/   Minimum/ % with                Percentiles
        Sample Size      Variance    Kurtosis    Maximum  Min/Max      20%/60%    40%/80%    Median

     X1                    0.000      -0.035      -3.139    0.20%      -0.842     -0.239     -0.016
             500.000       1.041       0.091       3.252    0.20%       0.254      0.887
     X3                   -0.067      -0.060      -3.145    0.20%      -0.870     -0.304     -0.034
             500.000       0.960       0.073       2.857    0.20%       0.205      0.741


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                        6

Loglikelihood

          H0 Value                        -433.426
          H0 Scaling Correction Factor      1.0174
            for MLR

Information Criteria

          Akaike (AIC)                     878.853
          Bayesian (BIC)                   904.140
          Sample-Size Adjusted BIC         885.096
            (n* = (n + 2) / 24)



MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 U1#1       ON
    X1                 0.769      0.165      4.669      0.000
    X3                 2.259      0.203     11.147      0.000

 U1#2       ON
    X1                 0.280      0.114      2.444      0.015
    X3                 0.885      0.143      6.200      0.000

 Intercepts
    U1#1              -0.749      0.158     -4.728      0.000
    U1#2               0.262      0.120      2.192      0.028


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.849E-01
       (ratio of smallest to largest eigenvalue)


LOGISTIC REGRESSION ODDS RATIO RESULTS

                                                95% C.I.
                    Estimate       S.E.  Lower 2.5% Upper 2.5%

 U1#1       ON
    X1                 2.157      0.355      1.562      2.978
    X3                 9.578      1.941      6.438     14.249

 U1#2       ON
    X1                 1.323      0.151      1.057      1.656
    X3                 2.423      0.346      1.832      3.206


STANDARDIZED MODEL RESULTS


STDYX Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 U1#1       ON
    X1                 0.784      0.170      4.619      0.000
    X3                 2.213      0.211     10.513      0.000

 U1#2       ON
    X1                 0.285      0.117      2.437      0.015
    X3                 0.867      0.143      6.084      0.000

 Intercepts
    U1#1              -0.749      0.158     -4.728      0.000
    U1#2               0.262      0.120      2.192      0.028


STDY Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 U1#1       ON
    X1                 0.769      0.165      4.669      0.000
    X3                 2.259      0.203     11.147      0.000

 U1#2       ON
    X1                 0.280      0.114      2.444      0.015
    X3                 0.885      0.143      6.200      0.000

 Intercepts
    U1#1              -0.749      0.158     -4.728      0.000
    U1#2               0.262      0.120      2.192      0.028


STD Standardization

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

 U1#1       ON
    X1                 0.769      0.165      4.669      0.000
    X3                 2.259      0.203     11.147      0.000

 U1#2       ON
    X1                 0.280      0.114      2.444      0.015
    X3                 0.885      0.143      6.200      0.000

 Intercepts
    U1#1              -0.749      0.158     -4.728      0.000
    U1#2               0.262      0.120      2.192      0.028


CONFIDENCE INTERVALS OF MODEL RESULTS

                  Lower .5%  Lower 2.5%    Lower 5%    Estimate    Upper 5%  Upper 2.5%   Upper .5%

 U1#1     ON
    X1               0.345       0.446       0.498       0.769       1.039       1.091       1.193
    X3               1.737       1.862       1.926       2.259       2.593       2.657       2.782

 U1#2     ON
    X1              -0.015       0.055       0.092       0.280       0.468       0.504       0.575
    X3               0.517       0.605       0.650       0.885       1.120       1.165       1.253

 Intercepts
    U1#1            -1.157      -1.059      -1.009      -0.749      -0.488      -0.438      -0.341
    U1#2            -0.046       0.028       0.065       0.262       0.459       0.497       0.570


CONFIDENCE INTERVALS FOR THE LOGISTIC REGRESSION ODDS RATIO RESULTS

 U1#1     ON
    X1               1.411       1.562       1.645       2.157       2.828       2.978       3.296
    X3               5.682       6.438       6.862       9.578      13.368      14.249      16.144

 U1#2     ON
    X1               0.985       1.057       1.096       1.323       1.597       1.656       1.777
    X3               1.678       1.832       1.916       2.423       3.065       3.206       3.500


CONFIDENCE INTERVALS OF STANDARDIZED MODEL RESULTS


STDYX Standardization

                  Lower .5%  Lower 2.5%    Lower 5%    Estimate    Upper 5%  Upper 2.5%   Upper .5%

 U1#1     ON
    X1               0.347       0.451       0.505       0.784       1.063       1.117       1.221
    X3               1.671       1.801       1.867       2.213       2.560       2.626       2.756

 U1#2     ON
    X1              -0.016       0.056       0.093       0.285       0.478       0.515       0.587
    X3               0.500       0.588       0.633       0.867       1.102       1.146       1.234

 Intercepts
    U1#1            -1.157      -1.059      -1.009      -0.749      -0.488      -0.438      -0.341
    U1#2            -0.046       0.028       0.065       0.262       0.459       0.497       0.570


STDY Standardization

                  Lower .5%  Lower 2.5%    Lower 5%    Estimate    Upper 5%  Upper 2.5%   Upper .5%

 U1#1     ON
    X1               0.345       0.446       0.498       0.769       1.039       1.091       1.193
    X3               1.737       1.862       1.926       2.259       2.593       2.657       2.782

 U1#2     ON
    X1              -0.015       0.055       0.092       0.280       0.468       0.504       0.575
    X3               0.517       0.605       0.650       0.885       1.120       1.165       1.253

 Intercepts
    U1#1            -1.157      -1.059      -1.009      -0.749      -0.488      -0.438      -0.341
    U1#2            -0.046       0.028       0.065       0.262       0.459       0.497       0.570


STD Standardization

                  Lower .5%  Lower 2.5%    Lower 5%    Estimate    Upper 5%  Upper 2.5%   Upper .5%

 U1#1     ON
    X1               0.345       0.446       0.498       0.769       1.039       1.091       1.193
    X3               1.737       1.862       1.926       2.259       2.593       2.657       2.782

 U1#2     ON
    X1              -0.015       0.055       0.092       0.280       0.468       0.504       0.575
    X3               0.517       0.605       0.650       0.885       1.120       1.165       1.253

 Intercepts
    U1#1            -1.157      -1.059      -1.009      -0.749      -0.488      -0.438      -0.341
    U1#2            -0.046       0.028       0.065       0.262       0.459       0.497       0.570


SAVEDATA INFORMATION


  Results in H5 Format

  Save file
    ex3.6.h5

  List of results saved

    Summary of Analysis
    Model Fit Information
    Model Results
    Logistic Regression Odds Ratio Results
    Standardized Model Results
    Confidence Intervals of Model Results
    Confidence Intervals of Standardized Model Results

     Beginning Time:  12:05:57
        Ending Time:  12:05:57
       Elapsed Time:  00:00:00



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2024 Muthen & Muthen
